sci-visualization/mantid/files/mantid-3.3.0-HAVE_IOSTREAM.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/Framework/Geometry/CMakeLists.txt b/Framework/Geometry/CMakeLists.txt
+index bb1d146..466b072 100644
+--- a/Framework/Geometry/CMakeLists.txt
++++ b/Framework/Geometry/CMakeLists.txt
+@@ -351,10 +351,8 @@ else ()
+   add_definitions ( -DNO_OPENCASCADE )
+ endif ()
+ 
+-# A few defines needed for OpenCascade on the Mac
+-if ( APPLE )
+-  add_definitions ( -DHAVE_IOSTREAM -DHAVE_LIMITS -DHAVE_IOMANIP )
+-endif ()
++# A few defines needed for OpenCascade
++add_definitions ( -DHAVE_IOSTREAM -DHAVE_LIMITS -DHAVE_IOMANIP )
```

Comment:
Good point, but whenever there is a chance to avoid patches, which needs porting with every release, by setting a couple of cmake variables we should go this way. And again it would be good to get these changes upstream.